requirements/requirements.txt

```diff
@@ -18,6 +18,7 @@ ovos-tts-plugin-mimic2~=0.1, >=0.1.5
 ovos-tts-plugin-google-tx~=0.0, >=0.0.3
 ovos-ww-plugin-pocketsphinx~=0.1
 ovos-ww-plugin-precise~=0.1
+ovos-ww-plugin-vosk~=0.1
```

Comment:
specify minimum version also, tests are failing this should fix them `ovos-ww-plugin-vosk~=0.1, >=0.1.1a1`